scripts/rabbit/setupRabbit.py

```diff
@@ -835,6 +849,19 @@ def setup(
     datagroups.fit_axes = fitvar
     datagroups.channel = channel
 
+    if args.selection:
+        for sel in args.selection:
+            sel_ax, sel_lb, sel_ub = sel.split()
+            sel_lb = complex(sel_lb)
```

Comment:
I think there is an issue for the "0" case which would be converted to a complex 0 and thus take the bin that is at 0 instead bin index 0. We had a similar issue before with axlim etc. What we should do instead is to use "sel_lb = parsing.str_to_complex_or_int(sel_lb)" and same for sel_ub. The the user can select bin by value via "--select 'ptll 0j 10j'" or by index via "--select 'ptll 0 10'".